test/controllers/concerns/parameters/nic_interface_test.rb

```diff
@@ -14,7 +14,6 @@ class NicInterfaceParametersTest < ActiveSupport::TestCase
 
     assert_equal 'test.example.com', filtered['name']
     assert_equal({'foo' => 'bar', 'memory' => 2}, filtered['compute_attributes'].to_h)
-    assert filtered.permitted?
```

Comment:
@xprazak2 just wondering - why remove this assertion?

Reply:
Because `filtered` are not strong params, just a regular hash, so `filtered.respond_to?(:permitted?) #=> false`. Since strong params no longer inherit from hash, we turn permitted params into [hash](https://github.com/theforeman/foreman/pull/5026/files#diff-21ec49eed71b67314a462a807396ef21)